tests/core/training/converters/test_story_markdown_to_yaml_converter.py

```diff
@@ -116,3 +116,75 @@ async def test_test_stories(tmpdir: Path):
             "  - action: respond_faq\n"
             "  - action: action_set_faq_slot\n"
         )
+
+
+async def test_test_stories_conversion_response_key(tmpdir: Path):
+    converted_data_folder = tmpdir / "converted_data"
+    os.mkdir(converted_data_folder)
+
+    test_data_folder = tmpdir / "tests"
+    os.makedirs(test_data_folder, exist_ok=True)
+    test_data_file = Path(test_data_folder / "test_stories.md")
+
+    simple_story_md = """
+    ## id
+
+    * out_of_scope/other: hahaha
+        - utter_out_of_scope/other
+    """
+
+    with open(test_data_file, "w") as f:
+        f.write(simple_story_md)
+
+    await StoryMarkdownToYamlConverter().convert_and_write(
+        test_data_file, converted_data_folder
+    )
+
+    assert len(os.listdir(converted_data_folder)) == 1
+    with open(f"{converted_data_folder}/test_stories_converted.yml", "r") as f:
+        content = f.read()
+        assert content == (
+            f'version: "{LATEST_TRAINING_DATA_FORMAT_VERSION}"\n'
+            "stories:\n"
+            "- story: id\n"
+            "  steps:\n"
+            "  - intent: out_of_scope/other\n"
+            "    user: |-\n"
+            "      hahaha\n"
+            "  - action: utter_out_of_scope/other\n"
+        )
```

Comment:
OPT: style suggestion, use multiline string instead (needs `import textwrap`)
```suggestion
        assert content == textwrap.dedent(f"""
            version: "{LATEST_TRAINING_DATA_FORMAT_VERSION}"
            stories:
            - story: id
              steps:
              - intent: out_of_scope/other
                user: |-
                  hahaha
              - action: utter_out_of_scope/other
          """)
```

Reply:
thank you!! since it already got merged, I will consider this for the next test I write!